plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -705,6 +712,9 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
         String deltaLogDirectory = getTransactionLogDir(location);
         Optional<Long> checkpointInterval = getCheckpointInterval(tableMetadata.getProperties());
         Optional<Boolean> changeDataFeedEnabled = getChangeDataFeedEnabled(tableMetadata.getProperties());
+        if (changeDataFeedEnabled.isPresent() && changeDataFeedEnabled.get()) {
+            checkChangeDataFeedColumnNames(tableMetadata.getColumns().stream().map(ColumnMetadata::getName).collect(Collectors.toSet()));
```

Comment:
Why not `.collect(toImmutableSet())`?

Reply:
nit: `checkChangeDataFeedColumnNames(List<ColumnMetadata> columns)`?

`.stream().map(ColumnMetadata::getName).collect(Collectors.toSet())` could be moved to method to reduce the burden on caller.